fastlane/Fastfile

```diff
@@ -7,7 +7,6 @@ platform :android do
 
     APK_LOCATION = "#{lane_context[SharedValues::GRADLE_APK_OUTPUT_PATH]}"
     sh("mv", "#{APK_LOCATION}", ENV["OUTPUT_PATH"])
-    sh("mv", ENV["OUTPUT_PATH"], "../")
```

Comment:
if we remove this line, Browserstack never finds the package, I don't understand why but that's why it's there

Reply:
I have changed the browserstack action file_path. Do you think this could work?